source/fonts.css

```diff
@@ -1,76 +1,54 @@
+         url('./fonts/creatodisplay-black-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-black-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-black-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
 }
 
 @font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Semibold.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Semibold'),
-		url('./fonts/ProximaNova-Semibold.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Semibold.woff') format('woff'),
-		url('./fonts/ProximaNova-Semibold.ttf') format('truetype');
-	font-weight: 600;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-medium-webfont.eot');
+    src: local('creato'), local('creatodisplay-medium-webfont'),
+         url('./fonts/creatodisplay-medium-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-medium-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-medium-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-medium-webfont.ttf') format('truetype');
+    font-weight: 600;
+    font-style: normal;
 }
 
 @font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Bold.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Bold'),
-		url('./fonts/ProximaNova-Bold.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Bold.woff') format('woff'),
-		url('./fonts/ProximaNova-Bold.ttf') format('truetype');
-	font-weight: bold;
-	font-style: normal;
+    font-family: 'creato';
+    src: url('./fonts/creatodisplay-regular-webfont.eot');
+    src: local('creato'), local('creatodisplay-regular-webfont'),
+         url('./fonts/creatodisplay-regular-webfont.eot?#iefix') format('embedded-opentype'),
+         url('./fonts/creatodisplay-regular-webfont.woff') format('woff'),
+         url('./fonts/creatodisplay-regular-webfont.woff2') format('woff2'),
+         url('./fonts/creatodisplay-regular-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
 }
 
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Extrabld.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Extrabld'),
-		url('./fonts/ProximaNova-Extrabld.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Extrabld.woff') format('woff'),
-		url('./fonts/ProximaNova-Extrabld.ttf') format('truetype');
-	font-weight: 800;
-	font-style: normal;
-}
 
-@font-face {
-	font-family: 'Proxima Nova';
-	src: url('./fonts/ProximaNova-Black.eot');
-	src: local('Proxima Nova'), local('ProximaNova-Black'),
-		url('./fonts/ProximaNova-Black.eot?#iefix') format('embedded-opentype'),
-		url('./fonts/ProximaNova-Black.woff') format('woff'),
-		url('./fonts/ProximaNova-Black.ttf') format('truetype');
-	font-weight: 900;
-	font-style: normal;
-}
```

Comment:
Is there a reason that creato extra bold and bold was left out?

Reply:
There's none. Just need a suggestion before adding it. Thank you.